config/tls/issuers/broker-ingress-issuer.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+  namespace: knative-eventing
+spec:
+  selfSigned: {}
+  
```

Comment:
aren't these all the same?

I'd potentially also change the name, to be a bit more explicit